src/scss/404.scss

```diff
@@ -0,0 +1,34 @@
+@keyframes swinging {
+  0% {
+    transform-origin: top right;
+    transform: rotate(0deg);
+  }
+  15% {
+    transform-origin: top right;
+    transform: rotate(-40deg);
+  }
+  30% {
+    transform-origin: top right;
+    transform: rotate(-20deg);
+  }
+  45% {
+    transform-origin: top right;
+    transform: rotate(-35deg);
+  }
+  60% {
+    transform-origin: top right;
+    transform: rotate(-25deg);
+  }
+  75% {
+    transform-origin: top right;
+    transform: rotate(-30deg);
+  }
+  to {
+    transform-origin: top right;
+    transform: rotate(-32deg);
+  }
+}
+
+.logo:hover {
```

Comment:
Maybe it happens not on `hover` but after a few seconds from the page load.. just once. I am not sure about repeating the animation each time the user hover the graphic... we could have also an a11y issue.